### PR TITLE
Feature/title drop=down refreshes

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/8749
https://www.openproject.org/work_packages/9297

### CODE REVIEW

**Comment by myabc** [comment]
@tilt why the `=` in the branch name?

**Comment by tilt** [comment]
@myabc because it's next to the `-`

File: app/assets/javascripts/angular/directives/components/toggled-multiselect-directive.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ angular.module('openproject.uiComponents')
       };
 
       scope.isSelected = function(value) {
-        return (Array.isArray(scope.values) && (scope.values.indexOf(value) !== -1) || scope.values.indexOf(value.toString()) !== -1);
+        return (Array.isArray(scope.values) && ((scope.values.indexOf(value) !== -1) || scope.values.indexOf(value.toString()) !== -1));
```
Comment (manwithtwowatches) [inline app/assets/javascripts/angular/directives/components/toggled-multiselect-directive.js:50]: @tilt Thanks for cleaning up my mess:D

**Comment by manwithtwowatches** [comment]
@tilt Probably outside the scope of this PR but worth mention the problem when you rename a query but then cancel and the name in the title remains the same.

**Pull request merged**